### PR TITLE
fix ios text input size

### PR DESCRIPTION
**Pull request by thesahindia**
### Details

Fixed the size of ios multiline TextInput

### Fixed Issues

$ https://github.com/Expensify/App/issues/7495

### Tests

- [x] Verify that no errors appear in the JS console

### QA Steps

1. Open ios app
2. Navigate to Settings >Security > Close account
3. Verify that the multiline TextInput is not different from other platforms
- [x] Verify that no errors appear in the JS console

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

Screenshot 2022-02-02 at 1 53 36 PM

#### Mobile Web
Screenshot 2022-02-02 at 1 51 18 PM

#### Desktop

Screenshot 2022-02-02 at 1 55 00 PM

#### iOS

Screenshot 2022-02-02 at 1 49 07 PM

#### Android

Screenshot 2022-02-02 at 1 49 51 PM

### CODE REVIEW

File: src/pages/settings/Security/CloseAccountPage.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ class CloseAccountPage extends Component {
                             value={this.state.reasonForLeaving}
                             onChangeText={reasonForLeaving => this.setState({reasonForLeaving})}
                             label={this.props.translate('closeAccountPage.enterMessageHere')}
-                            containerStyles={[styles.mt5]}
+                            containerStyles={[styles.mt5, styles.closeAccountMessage]}
```
Comment (luacmartins) [inline src/pages/settings/Security/CloseAccountPage.js:79]: On second thought, maybe we should name this `closeAccountMessageInput` so it's a bit more descriptive.

**Review by rushatgabhane — DISMISSED** [review]
@luacmartins Tests well on all platforms, LGTM!

File: src/pages/settings/Security/CloseAccountPage.js (hunk 1 above)
Comment (thesahindia) [reply src/pages/settings/Security/CloseAccountPage.js]: Done.

**Review by luacmartins — APPROVED** [review]
LGTM! Thanks for the changes @thesahindia!

**Pull request merged**